calico_containers/pycalico/block.py

```diff
@@ -543,3 +543,9 @@ class AddressNotAssignedError(BlockError):
     Tried to query an address that isn't assigned.
     """
     pass
+
+class AddressRangeNotAllowedError(BlockError):
```

Comment:
Not sure I like the name of this.  It isn't a range. it's a CIDR.

Perhaps just CIDRToSmallError or something along those lines.